ci/tasks/build-stemcell.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+trap clean_vagrant EXIT
+
+set_up_vagrant_private_key() {
+  key_path=$(mktemp -d /tmp/ssh_key.XXXXXXXXXX)/value
+  echo "$BOSH_PRIVATE_KEY" > $key_path
+  chmod 600 $key_path
+  export BOSH_VAGRANT_KEY_PATH=$key_path
+  eval `ssh-agent`
+  ssh-add $key_path
+}
+
+clean_vagrant() {
+  vagrant destroy remote -f || true
+}
+
+get_ip_from_vagrant_ssh_config() {
+  config=$(vagrant ssh-config remote)
+  echo $(echo "$config" | grep HostName | awk '{print $2}')
+}
+
+build_num=$(cat stemcell-version/number | sed 's/\.0$//;s/\.0$//')
+
+pushd bosh-src
+
+# todo check out correct version of bosh-src for that stemcell
```

Comment:
Removed these comments, which should be left by Kara, we don't see any problem without that part of code.